### PR TITLE
[akeneo-design-system]: Moving to babel loader

### PR DESCRIPTION
**Pull request by nicomaligne**
**Description (for Contributor and Core Developer)**
POC in progress, here we are changing typescript loader to babel loader with typescript plugins.

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | -
| Added legacy Behats               | -
| Added acceptance tests            | -
| Added integration tests           | -
| Changelog updated                 | -
| Review and 2 GTM                  | -
| Micro Demo to the PO (Story only) | -
| Migration script                  | -
| Tech Doc                          | -

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

File: akeneo-design-system/tsconfig.json [hunk 1]
```diff
@@ -26,6 +25,4 @@
         "suppressImplicitAnyIndexErrors": true,
         "target": "es5"
     },
-    "include": ["src/**/*"],
-    "exclude": ["node_modules"]
```
Comment (pchasle) [inline akeneo-design-system/tsconfig.json]: I think that we still need the include/exclude for development / IDE

File: akeneo-design-system/tsconfig.json [hunk 2]
```diff
@@ -17,7 +14,9 @@
         "noImplicitThis": true,
         "noUnusedLocals": true,
         "noUnusedParameters": true,
+        "outDir": "lib",
         "rootDir": "src",
+        "skipLibCheck": true,
```
Comment (pchasle) [inline akeneo-design-system/tsconfig.json]: I wouldn't use the flag unless we effectively have errors in some malformed libs/dt.ts and/or this improve significantly the build time.

Comment (nicomaligne) [reply akeneo-design-system/tsconfig.json]: Yep I need to investigate on this, I've got error when I build on the pim project but not on a test one. Need to check dependencies.

File: akeneo-design-system/tsconfig.json (hunk 1 above)
Comment (nicomaligne) [reply akeneo-design-system/tsconfig.json]: I think with the rootDir do the trick.

File: akeneo-design-system/tsconfig.json (hunk 2 above)
Comment (pchasle) [reply akeneo-design-system/tsconfig.json]: Keep it this way for now then ;)

**Pull request merged**